src/bundle.rs

```diff
@@ -108,6 +108,7 @@ impl Flags {
     ///
     /// [txencoding]: https://zips.z.cash/protocol/protocol.pdf#txnencoding
     pub fn from_byte(value: u8) -> Option<Self> {
+        // TCR:bad-txns-v5-reserved-bits-nonzero
```

Comment:
```suggestion
        // https://p.z.cash/TCR:bad-txns-v5-reserved-bits-nonzero
```
(see #321 for other examples)